temporalio/client.py

```diff
@@ -702,6 +750,11 @@ async def result(self, *, follow_runs: bool = True) -> ReturnType:
             follow_runs: If true (default), workflow runs will be continually
                 fetched, until the most recent one is found. If false, the first
                 result is used.
+            rpc_metadata: Headers used on each RPC call. Keys here are always
+                overridden by client-level RPC metadata keys.
+            rpc_timeout: Optional RPC deadline to set for each RPC call. Note,
+                this is the timeout for each history RPC call not this overall
+                function.
```

Comment:
People won't realize what "history" means in this context. I'd just say "...underlying RPC call...".

Reply:
Will change on next commit. I am a bit worried about confusion caused by this and if I should change to a fixed date time deadline, but I can wait until that is really needed.